tests/conftest.py

```
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: MPL-2.0
+#
+# conftest.py -- top-level pytest configuration
+# Copyright (C) 2025  Jacob Koziej <jacobkoziej@gmail.com>
+
+import pytest
+
+from typing import Final
+
+from pytest import FixtureRequest
+
+
+RANDOM_COUNT: Final[int] = 128
+
+
+def _seed_id(param: int):
+    return f"seed=0x{param:08x}"
+
+
+@pytest.fixture(
+    ids=_seed_id,
+    params=[
+        0x626F6272,
```

Is this the random seed specific to the entire suite of tests at the moment?

---

Yes, we still need to add an option to `pytest` to default to a random seed for each call of the test suite, but for now this will be the list of seeds we use for testing.